### PR TITLE
Manage multiple listings

### LINKED ISSUE #107: Update multiple listings status
## Description
As a registered user\
I want to edit multiple listing status\
so that I can indicate product's availability

## Acceptance Criteria
- User must be logged in
- User can only edit listings that they own

## Dev tasks
- [x] Will use same component as story `View all own listings`
- [x] [Create endpoint to apply changes to all selected listings](https://github.com/gilhrpenner/COMP4350/issues/98)

Story points: 1/5

### PR DESCRIPTION
**Pull request by gilhrpenner**
# What I did
- Changed some `listingRequests` function names
- Created new function to delete multiple listings by id
- Endpoint to update listings status
- Some extra translations

# To do
- The frontend stuff to delete multiple listings is ready but needs to hook up with the backend

# Preview

https://user-images.githubusercontent.com/24423300/158740804-9375f4ce-48ea-4dc8-b338-d5cae4426bfb.mov

Closes #107

### CODE REVIEW

**Review by MarkRobitaille — APPROVED** [review]
Looks good to me. 👍

File: frontend/src/pages/dashboard/Dashboard.vue [hunk 1]
```diff
@@ -114,17 +168,86 @@ const deleteListing = (listingId: string) => {
...
+    },
+];
+
+const onDeleteSelectedListings = () => {
+    const ids = selected.value.map(listing => listing.id);
+    deleteListingsRequest(ids);
+};
+
+const onUpdateListingsStatus = async () => {
+    if (!listingStatus.value || !selected.value.length) return;
```
Comment (MarkRobitaille) [inline frontend/src/pages/dashboard/Dashboard.vue:210]: While testing this I was curious if it should return to being null if there are no listings selected? If you select a listing status and then select some listings, you can't update the listing status without changing out to a different status.

Comment (gilhrpenner) [reply frontend/src/pages/dashboard/Dashboard.vue:210]: Absolutely, I can easily fix that!

Comment (gilhrpenner) [reply frontend/src/pages/dashboard/Dashboard.vue]: Fixed! :)

**Pull request merged**